### PR TITLE
feat: docusaurus homepage

### PR DESCRIPTION
**Pull request by carlagn**
## Describe this PR

Docusaurus homepage updated to the current live layout. 

## Changes

Update theming and the index page.

## What issue does this fix?

https://linear.app/prisma-company/issue/DA-65/docs-homepage

## Any other relevant information

- [x] responsive
- [x] dark/light mode

### CODE REVIEW

File: config.ts [hunk 1]
```diff
@@ -0,0 +1,180 @@
+const siteConfigData = {
```
Comment (jharrell) [inline config.ts:1]: Is there a way we can do away with this file? I know it's a bit of extra work but doing something [like this](https://github.com/prisma/docs/blob/docusaurus/src/components/HomepageFeatures/index.tsx) feels a bit better to me.

File: config.ts [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    ],
+    CommunityLinksData: [
+      // {
+      //   id: 'slack',
+      //   title: 'Slack',
+      //   description: 'Engage with a vibrant community of developers and companies adopting Prisma',
+      //   icon: 'fa-brands fa-slack',
+      //   link: 'https://slack.prisma.io/',
+      //   linkText: 'Join our channel',
+      // },
```
Comment (jharrell) [inline config.ts:107]:
```suggestion
```

File: src/components/icons/technologies/CockroachDB.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import * as React from 'react'
```
Comment (jharrell) [inline src/components/icons/technologies/CockroachDB.tsx:1]: Is there a reason why we need these as TSX components rather than just svgs?

**Review by jharrell — APPROVED** [review]
If it works, it works, but I'd like to see a few changes. Happy to take those on myself to free you up.

File: config.ts (hunk 1 above)
Comment (carlagn) [reply config.ts:1]: To have content + logic on the same file? sure!

**Pull request merged**